ddtrace/encoding.py

```diff
@@ -9,11 +9,13 @@
 try:
     from ddtrace.vendor import msgpack
     # DEV: `use_bin_type` only exists since `0.4.0`, but we vendor a more recent version
-    MSGPACK_PARAMS = {'use_bin_type': True}
```

Comment:
So that was unused right?

I feel like fixing that would deserve a different PR, but ok.